app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -85,6 +86,7 @@ class ConversationViewModel @Inject constructor(
                         (message.content as? MessageContent.Text)?.value ?: "content is not available"
                     )
                 ),
+                messageSource= MessageSource.CurrentUser,
```

Comment:
Just a small formatting nitpicking
```suggestion
                messageSource = MessageSource.CurrentUser,
```